raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -210,13 +211,13 @@ scenario:
           name: "Make 100 payments from 3 to 0"
           repeat: 100
           tasks:
-            - transfer: {from: 3, to: 0, amount: 1}
+            - transfer: {from: 3, to: 0, amount: 100}
       - serial:
           name: "Assert after 100 payments from 3 to 0"
           tasks:
             - wait: 10
-            - assert_sum: {from: 0, balance_sum: 1995}
-            - assert_sum: {from: 3, balance_sum: 2000}
+            - assert_sum: {from: 0, balance_sum: 205_000}
```

Comment:
why did these number change compared to the old one? Didn't you want to multiply everything with 100?